manifests/j/Jidifudi/GT4T/8.49.250305/Jidifudi.GT4T.installer.yaml

```
@@ -0,0 +1,30 @@
+# Automatically updated by the winget bot at 2025/Mar/06
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.10.0.schema.json
+
+PackageIdentifier: Jidifudi.GT4T
+PackageVersion: 8.49.250305
+MinimumOSVersion: 10.0.0.0
+InstallerType: inno
+Scope: machine
+UpgradeBehavior: install
+Installers:
+- Architecture: x86
+  InstallerUrl: https://gt4t.net/apps/gt4t_win_32_bit/GT4T_setup.exe
+  InstallerSha256: 9B160A0ED89D527A745DE797915E0E3D2C04E9C2FCF86DC18EBB002C65448931
+  ProductCode: '{8EC2D6A4-7B61-4737-8DA2-B6D4339EE744}_is1'
+- Architecture: x64
+  InstallerUrl: https://gt4t.net/apps/gt4t_win_64_bit/GT4T_setup.exe
+  InstallerSha256: bda2394f56ff927dbe34347c00dbb54ef68f3b1a8c5300e6c649f31d8384db21
+  ProductCode: '{8EC2D6A4-7B61-4737-8DA2-B6D4339EE744}_is1'
+- InstallerLocale: zh-CN
+  Architecture: x86
+  InstallerUrl: https://dl.gt4t.cn/downloads/apps/gt4t_win_32_bit/GT4T_setup.exe
+  InstallerSha256: 9B160A0ED89D527A745DE797915E0E3D2C04E9C2FCF86DC18EBB002C65448931
+  ProductCode: '{8EC2D6A4-7B61-4737-8DA2-B6D4339EE744}_is1'
+- InstallerLocale: zh-CN
+  Architecture: x64
+  InstallerUrl: https://dl.gt4t.cn/downloads/apps/gt4t_win_64_bit/GT4T_setup.exe
+  InstallerSha256: bda2394f56ff927dbe34347c00dbb54ef68f3b1a8c5300e6c649f31d8384db21

```

```suggestion
  InstallerSha256: 67229D38D3B4FE5D8BE405F23073F6559636EA2C7C2810194ACD88C0330A300F
```

(Automated response - build 994.)